myapp.rb

```diff
@@ -6,6 +6,7 @@
 require './models/message'
 require './models/user'
 require './models/follower'
+require 'newrelic_rpm'
```

Comment:
Is this necessary if it is not being used directly in this code?
I don't see any uses of `require` in this installation guide:
https://newrelic.com/ruby#toc--how-do-i-install-the-new-relic-agent-

Reply:
https://docs.newrelic.com/docs/apm/agents/ruby-agent/instrumented-gems/sinatra-instrumentation/